lib/Models/Terria.ts

```diff
@@ -320,6 +320,8 @@ interface ConfigParameters {
   googleAnalyticsOptions?: unknown;
 
   relatedMaps?: RelatedMap[];
+
+  aboutButtonHrefUrl?: string | null;
```

Comment:
Any reason for using `null` instead of `undefined`?

Reply:
`undefined` is not a valid JSON value. So we need `null` to be able to "un-set" the default value of `aboutButtonHrefUrl`